content/workflows/csp/aws/service/timestream/resources/database/policy-context/timestream-db-not-encrypted-with-cmk-via-kms.yaml

```
@@ -0,0 +1,27 @@
+schema: 1
+type: policy-context
+
+id: timestream-db-not-encrypted-with-cmk-via-kms
```

```suggestion
id: timestream-db-is-not-encrypted-with-cmk-via-kms
```

Rename file as well.